espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -57,6 +58,26 @@ def add_arguments(parser):
                            help='optimizer warmup steps')
         group.add_argument('--transformer-length-normalized-loss', default=True, type=strtobool,
                            help='normalize loss by length')
+
+        E2E_rnn.loss_add_arguments(parser)
+        E2E_rnn.recognition_add_arguments(parser)
```

Comment:
I think this `add_arguments` might not be defined in `E2E(_rnn)` because its option can be applied to any models.
How about move this to `ASRInterface`?

Reply:
I'm not sure which is better in the current status. 
Maybe, if always it has these option, It's better to put it back to espnet/bin/asr_train.py,
because Interface is not allowed to have any concrete methods, strictly speaking.

Is is True? These option assumes the beam search based decoding.
Hmm, I'll follow you. Please decide.

Reply:
I got it. Let's put it back to espnet/bin/asr_train.py.

Reply:
o.k.